Gemeinsame Unterverzeichnisse: debootstrap-0.2.15.orig/debian und debootstrap-0.2.15/debian.
Gemeinsame Unterverzeichnisse: debootstrap-0.2.15.orig/po und debootstrap-0.2.15/po.
diff -u debootstrap-0.2.15.orig/sarge debootstrap-0.2.15/sarge
--- debootstrap-0.2.15.orig/sarge	2003-11-01 12:56:54.000000000 +0100
+++ debootstrap-0.2.15/sarge	2003-11-04 00:46:59.000000000 +0100
@@ -43,7 +43,7 @@
 	    IPFWTOOL="ipchains"
             ;;
         "powerpc")
-            base="$base quik mac-fdisk amiga-fdisk psmisc powerpc-utils setserial pciutils hfsutils yaboot pcmcia-cs"
+            base="$base quik mac-fdisk amiga-fdisk psmisc powerpc-utils setserial pciutils hfsutils yaboot pcmcia-cs libreadline4"
 	    IPFWTOOL="ipchains iptables"
             ;;
         "sparc")
diff -u debootstrap-0.2.15.orig/sid debootstrap-0.2.15/sid
--- debootstrap-0.2.15.orig/sid	2003-11-01 12:56:54.000000000 +0100
+++ debootstrap-0.2.15/sid	2003-11-04 00:45:59.000000000 +0100
@@ -41,7 +41,7 @@
 	    IPFWTOOL="ipchains"
             ;;
         "powerpc")
-            base="$base quik mac-fdisk amiga-fdisk psmisc powerpc-utils setserial pciutils hfsutils yaboot pcmcia-cs"
+            base="$base quik mac-fdisk amiga-fdisk psmisc powerpc-utils setserial pciutils hfsutils yaboot pcmcia-cs libreadline4"
 	    IPFWTOOL="ipchains iptables"
             ;;
         "sparc")
